src/passes/Print.cpp

```diff
@@ -454,7 +454,7 @@ struct PrintSExpression : public Visitor<PrintSExpression> {
     if (curr->params.size() > 0) {
       for (size_t i = 0; i < curr->params.size(); i++) {
```

Comment:
There is another option though, of `params` being a type that has `size()` returning `Index`, instead of an `std::vector`. (Perhaps an `IndexedVector`?) Anyhow, just a thought, maybe it's a bad idea, but I also think lots of toIndex calls is bad as well.

The only think toIndex saves us from is if the number of params is more than can fit in a 32-bit value? I don't think that risk is worth the cast. I *do* think the cast is useful when receiving outside input, like when parsing s-expressions. But internally, like here, it feels like just clutter.